cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -241,6 +245,7 @@
     {% for feature, sources, targets in feature_reos %}
         {% if forloop.first %}
             <div style="display: flex; justify-content: space-between">
+                {% if feature.name %}
```

Comment:
I think both SVGs are the same, so instead of structuring your changes like 
```
if feature.name
  <h1>
     <svg></svg>
     [feature name]
  </h1>
else
   <h1>
     <svg></svg>
     [feature type] (location)
  </h1>
endif 
```
You should do 
```
   <h1>
     <svg></svg>
if feature.name
     [feature name]
else
     [feature type] (location)
endif
  </h1> 
```